files/ko/web/http/headers/referer/index.html

```diff
@@ -1,40 +1,43 @@
 ---
 title: Referer
 slug: Web/HTTP/Headers/Referer
+tags:
+  - HTTP
+  - Reference
+  - header
+  - referer
+  - referrer
 translation_of: Web/HTTP/Headers/Referer
+browser-compat: http.headers.Referer
 ---
 <div>{{HTTPSidebar}}</div>
 
-<p><code><strong>Referer</strong></code> 요청 헤더는 현재 요청된 페이지의 링크 이전의 웹 페이지 주소를 포함합니다. <code>Referer</code> 헤더는 사람들이 어디로부터 와서 방문 중인지를 인식할 수 있도록 해주며 해당 데이터는 예를 들어, 분석, 로깅, 혹은 캐싱 최적화에 사용될 수도 있습니다.</p>
+<p><code><strong>Referer</strong></code> 요청 헤더는 현재 요청된 페이지의 링크 이전의 웹 페이지 주소를 포함합니다. <code>Referer</code> 헤더는 사람들이 어디로부터 와서 방문 중인지를 인식할 수 있도록 해주며 해당 데이터는 예를 들어, 분석, 로깅, 혹은 캐싱 최적화에 사용될 수도 있습니다.</p>
 
-<p>referer는 실제로 단어 "referrer"에서 철자를 빼먹은 것입니다. 자세한 내용은 {{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}을 참고하세요.</p>
+<p><code>Referer</code> 헤더는 URL 프래그먼트 (예 : "#section") 또는 "username : password" 정보를 포함 할 수 없습니다. 잠재적으로 <em>원본</em>, <em>경로</em>, 및 <em>쿼리 문자열</em>을 포함 할 수 있습니다. 전송되는 내용은 요청에 대한 referer 정책에 따라 다릅니다. 정보 및 예제는 Referrer-Policy를 참조하십시오. <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#directives">정보</a> 와 <a href="/en-US/docs/Web/HTTP/Headers/Referrer-Policy#examples">예시</a>는 이곳 {{HTTPHeader("Referrer-Policy")}}을 참고하세요.
 
-<div class="warning">
-<p><code>Referer</code> 헤더는 사생활과 관련된 브라우징 히스토리에 관한 정보를 노출할 가능성이 있습니다.</p>
+<div class="notecard note">
+    <h4>Note</h4>
+    <p><p>referer는 실제로 단어 "referrer"에서 철자를 뺀 것입니다. 자세한 내용은 {{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}을 참고하세요.</p>
```

Comment:
misspelling은 잘못된 철자, 표기법을 의미하는 단어이기에 철자를 뺐다고 하면 뉘앙스가 다를 수 있을 거 같아 제안 드려봅니다 :)

```suggestion
    <p><p>referer는 단어 "referrer"의 잘못된 철자입니다. 자세한 내용은 {{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}을 참고하세요.</p>
```